cmd/admin-handlers.go

```diff
@@ -2704,7 +2704,7 @@ func appendClusterMetaInfoToZip(ctx context.Context, zipWriter *zip.Writer) {
 	go func() {
 		ci := clusterInfo{}
 		ci.Info.PoolsCount = len(globalEndpoints)
-		ci.Info.ServersCount = globalEndpoints.NEndpoints()
+		ci.Info.ServersCount = len(globalNotificationSys.ServerInfo()) + 1
```

Comment:
Can you fix the comment of NEndpoints(), also I guess you can use globalEndpoints.Hostnames() insted of this, it seems to be faster

Reply:
If you need unique peers you should use global callers instead of making a large request such as ServerInfo() to calculate number of servers.

Reply:
Thanks, done.